package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "o2r-subsitute",
+  "name": "o2r-substitute",
   "version": "0.1.0",
```

Comment:
Bump the minor version number, at least to `0.2.0`, because this PR includes many changes.